src/web-components/comments/components/annotation-pin.ts

```diff
@@ -1,16 +1,74 @@
-import { CSSResultGroup, LitElement, html } from 'lit';
+import { CSSResultGroup, LitElement, PropertyValueMap, html } from 'lit';
```

Comment:
PropertyValueMap is not been used
```suggestion
import { CSSResultGroup, LitElement, html } from 'lit';
```